app/src/main/java/edu/mills/findeatfood/RecipeDatabaseUtilities.java

```diff
@@ -0,0 +1,91 @@
+package edu.mills.findeatfood;
+
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+/**
+ * Provides an interface that connects to the database. Activities and fragments should
+ * make calls here instead of trying to connect directly to the database.
+ */
+public final class RecipeDatabaseUtilities {
+
+    private static final String RECIPE_TABLE = "FAVORITE_RECIPES";
+
+    private RecipeDatabaseUtilities() {
+    }
+
+    /**
+     * Inserts the given recipe into the database by using the SQLiteRecipeOpenHelper.
+     * @param db    Our SQLiteDatabase, where we will be inserting the recipe into.
+     * @param recipeName    Name of the recipe.
+     * @param recipeId  Id of the recipe.
+     * It inserts the RECIPE_TABLE into the database with the values of the
+     * name of the recipe and its id.
+     */
+    public static void insertRecipe(SQLiteDatabase db,
+                                    String recipeName,
+                                    String recipeId) {
+        ContentValues values = new ContentValues();
+        values.put(SQLiteRecipeOpenHelper.RECIPE_NAME_COL, recipeName);
+        values.put(SQLiteRecipeOpenHelper.RECIPE_ID_COL, recipeId);
+        db.insert(RECIPE_TABLE, null, values);
+    }
+
+    /**
+     * Gets all of the recipes in the database by using SQLiteRecipeOpenHelper.
```

Comment:
You might want to review the javadoc for `MillsGo.DatabaseUtilities` for correct javadoc style.